### DISCUSSION TITLE
[ZK Stack] -  Transactions are in queue

### DISCUSSION THREAD

**Discussion by devudilip** [zkSync-Community-Hub/zksync-developers#373] [ZK Stack] [answered]
### Module Affected

- [ ] System contracts (era-system-contracts)
- [ ] Smart contracts (era-contracts)
- [ ] zkEVM (era-zk_evm)
- [X] Server (zksync-era)
- [ ] Prover (era-boojum)
- [ ] Fee withdrawer (era-fee-withdrawer)
- [ ] Block Explorer (clock-explorer)
- [ ] Portal - Wallet + Bridge (dapp-portal)
- [ ] Other (please specify below)

### Rust Version

1.74.0

### Operating System

- [ ] Ubuntu
- [X] MacOS
- [ ] Windows
- [ ] Other (Please specify below)

### Issue Description

Ran ZK stack init with Sepolia as base layer. 
Was able to bridge the amount from Base layer(sepolia) to Zk Era Layer 2. 
But no transactions are working on the layer 2. 
Every transfers are in queue for long time.

### Expected Behavior

After transferring an amount from 1 account to another, should have deducted the amount from acc1 and deposited to acc2.
And Transactions should be executed successfully 

### Current Behavior

Very first transfer is in queue, and any following transfer is also in the queue for long time. 
Transactions are not executed.

### Additional Details

Running the server using  zk server --components "http_api,eth,state_keeper,housekeeper"

### Prior Research

- [X] Yes, I searched for similar issues and did not find any matching ones.

**Comment by devudilip** [comment, marked as answer]
running the below server command with components solved the issue. 

`zk server --components "http_api,eth,state_keeper,housekeeper,tree"
`
I was missing the 'tree component'

**Comment by stennettmeisha** [comment]
Transaction processing stalled on zkSync Sepolia Layer 2 after successful bridging from the base layer. Transactions remain queued without execution. Running server components include HTTP API, Ethereum integration, state keeper, and housekeeper. Further investigation required.

**Comment by propmacboorsp** [comment]
It appears that there may be an issue with transaction processing on the zkSync Layer 2 network after transferring funds from the Sepolia base layer. Transactions are getting stuck in the queue and not being executed promptly. This could be due to various reasons such as network congestion, resource limitations, or bugs in the system. To address this, it's recommended to check the logs for any error messages, monitor system resources, and consider reaching out to the zkSync community or developers for assistance in debugging and resolving the issue.

**Comment by erinemsyntracy** [comment]
The user ran zk stack init with Sepolia as the base layer and was able to bridge an amount from the base layer to Zk Era Layer 2. However, transactions on Layer 2 are not working, and every transfer is stuck in the queue for a long time without being executed. The user is running the server using zk server --components "http_api,eth,state_keeper,housekeeper". Without further information or access to the code repository, it is difficult to diagnose the exact issue.
